### PR TITLE
Pin r1.7 HF and fix nightly HF GLUE task

### PR DESCRIPTION
**Pull request by jysohn23**
On PT/XLA examples repo I've also updated the pin to HF dep to their `v3.4.0` release.

### CODE REVIEW

File: tests/pytorch/nightly/hf-glue.libsonnet [hunk 1]
```diff
@@ -22,12 +22,12 @@ local utils = import "templates/utils.libsonnet";
...
+    pip install datasets
     python examples/xla_spawn.py \
       --num_cores 8 \
       examples/text-classification/run_glue.py \
       --logging_dir=./tensorboard-metrics \
       --task_name MNLI \
-      --data_dir /datasets/glue/MNLI \
       --cache_dir ./cache_dir \
       --do_train \
       --do_eval \
```
Comment (zcain117) [inline tests/pytorch/nightly/hf-glue.libsonnet:33]: could you also remove from down below:

```
    volumeMap+: {
      datasets: common.datasetsVolume,
    },
```

File: tests/pytorch/r1.7/hf-glue.libsonnet [hunk 2]
```diff
@@ -19,8 +19,8 @@ local utils = import "templates/utils.libsonnet";
 
 {
   local command_common = |||
-    git clone https://github.com/huggingface/transformers.git
-    cd transformers && pip install .
+    git clone --recursive -b r1.7 https://github.com/pytorch-tpu/examples.git
+    cd examples/deps/transformers && pip install .
     git log -1
     python examples/xla_spawn.py \
       --num_cores 8 \
```
Comment (zcain117) [inline tests/pytorch/r1.7/hf-glue.libsonnet:26]: could you also remove from down below:

```
    volumeMap+: {
      datasets: common.datasetsVolume,
    },
```

**Review by zcain117 — CHANGES_REQUESTED** [review]
Thanks for this fix!

File: tests/pytorch/nightly/hf-glue.libsonnet (hunk 1 above)
Comment (jysohn23) [reply tests/pytorch/nightly/hf-glue.libsonnet:33]: Good point, done.

File: tests/pytorch/r1.7/hf-glue.libsonnet (hunk 2 above)
Comment (jysohn23) [reply tests/pytorch/r1.7/hf-glue.libsonnet:26]: `r1.7` depends on `v3.4.0` (latest stable) which still doesn't have the latest on demand GLUE dataset download change so we need the volume.

**Review by zcain117 — APPROVED** [review]

**Pull request merged**